### PR TITLE
Completed two-two-column-layout | Matt Powers

### PR DESCRIPTION
**Pull request by mpowers1113**
git stat
Screen Shot 2021-10-11 at 9 40 32 AM
us

### CODE REVIEW

File: css-two-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  <div class="container">
+    <div class="row-header">
+      <div class="column-full">
+        <h1>SCALE, SPEED, AGILITY</h1>
+        <p>We've dedicated thousands of man hours to making healthcare institutions in the Helium network run 10x more
+          efficiently and deliver and unparalleled quality of care.</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-half">
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:24]: This `column-one-half` class has more `column` elements in it. There will never be a `column` directly inside another `column`, this means you need another `row` inside this `column`

**Comment by mpowers1113** [comment]
Screen Shot 2021-10-11 at 8 24 55 PM

File: css-two-column-layout/index.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        </div>
+        <div class="row">
+          <img src="images/data.svg" alt="data" class="image">
+        </div>
+        <div class="row">
+          <h4>INTUITIVE, SMART SOFTWARE</h4>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Earum ad distinctio culpa saepe doloribus libero
+            quos
+            corrupti nulla iste similique!</p>
+        </div>
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:42]: Now you have multiple `row` elements, but no `column` elements 🤔 

For this layout, you'll need a `row` inside each `column-half` and two `column` class elements in that `row`, one for the image, and one for the text

File: css-two-column-layout/index.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        <div class="column-three-fourths">
+          <h4>INTUITIVE, SMART SOFTWARE</h4>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Earum ad distinctio culpa saepe doloribus libero
+            quos
+            corrupti nulla iste similique!</p>
+        </div>
+      </div>
+    </div>
+    <div class="button-row">
```
[the remark this answers is not in the document]
Comment (codingcodymiller) [reply css-two-column-layout/index.html:65]: This still needs to be addressed

**Comment by mpowers1113** [comment]
Thank you!

**Pull request merged**